llm4pddl/utils.py

```diff
@@ -176,6 +176,33 @@ def get_custom_task(benchmark_name: str, task_num: int) -> Task:
     return Task(domain_file, problem_file)
 
 
+def prep_problem(problem: str) -> str:
+    """Maps a string of a problem file to a new string.
+
+    This new string has had all space removed between grounded atoms.
```

Comment:
add the other things that you described in the PR description to this docstring too